__test__/inject.test.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+import { mount } from 'enzyme';
+import compose from '../compose';
+import inject from '../inject';
+
+describe('pure', () => {
+  class Title extends Component {
+    render () {
+      return (
+        <h1>{this.props.text}</h1>
```

Comment:
yeah I just try to rewrite the injectIntl from react-intl, the goal is to reproduce the "performance" issue between pure component and injectIntl.